lighthouse-core/formatters/table.js

```diff
@@ -78,13 +78,18 @@ class Table extends Formatter {
 
     const rows = results.map(result => {
       const cols = headingKeys.map(key => {
+        let value = result[key];
+        if (typeof value === 'undefined') {
```

Comment:
I found it super helpful for the CLI to differentiate between missing cells and (totally subjectively) more pleasing in the HTML, but open to other ideas